src/store/rescan.js

```diff
@@ -3,19 +3,23 @@ import { show, start } from "../api/rescan";
 export default {
   namespaced: true,
   state: {
-    rescan: null
+    rescan: null,
+    lastClick: new Date(0)
   },
   mutations: {
+    setLastClick(state, payload) {
+      state.lastClick = payload;
+    },
     setRescan(state, payload) {
-      state.rescan = payload;
+      state.rescan = Object.assign({}, state.rescan, payload);
```

Comment:
I don't think this change is necessary anymore? Correct me if I'm wrong.

Reply:
Thought we still needed this for reactivity, but apparently not